src/Ocelot.Provider.Polly/OcelotBuilderExtensions.cs

```diff
@@ -1,115 +1,209 @@
-    /// <param name="builder">Ocelot builder to extend.</param>
-    /// <param name="delegatingHandler">Your customized delegating handler (to manage QoS behavior by yourself).</param>
-    /// <returns>The reference to the same extended <see cref="IOcelotBuilder"/> object.</returns>
-    public static IOcelotBuilder AddPolly<T>(this IOcelotBuilder builder, QosDelegatingHandlerDelegate delegatingHandler)
-        where T : class, IPollyQoSProvider<HttpResponseMessage> =>
-        AddPolly<T>(builder, delegatingHandler, DefaultErrorMapping);
-
-    /// <summary>
-    /// Adds Polly QoS provider to Ocelot by defaults.
-    /// </summary>
-    /// <remarks>
-    /// Defaults:
-    /// <list type="bullet">
-    ///   <item><see cref="DefaultDelegatingHandler"/></item>
-    ///   <item><see cref="DefaultErrorMapping"/></item>
-    /// </list>
-    /// </remarks>
-    /// <typeparam name="T">QoS provider to use (by default use <see cref="PollyQoSProvider"/>).</typeparam>
-    /// <param name="builder">Ocelot builder to extend.</param>
-    /// <returns>The reference to the same extended <see cref="IOcelotBuilder"/> object.</returns>
-    public static IOcelotBuilder AddPolly<T>(this IOcelotBuilder builder)
-        where T : class, IPollyQoSProvider<HttpResponseMessage> =>
-        AddPolly<T>(builder, DefaultDelegatingHandler, DefaultErrorMapping);
-
-    /// <summary>
-    /// Adds Polly QoS provider to Ocelot by defaults with default QoS provider.
-    /// </summary>
-    /// <remarks>
-    /// Defaults:
-    /// <list type="bullet">
-    ///   <item><see cref="PollyQoSProvider"/></item>
-    ///   <item><see cref="DefaultDelegatingHandler"/></item>
-    ///   <item><see cref="DefaultErrorMapping"/></item>
-    /// </list>
-    /// </remarks>
-    /// <param name="builder">Ocelot builder to extend.</param>
-    /// <returns>The reference to the same extended <see cref="IOcelotBuilder"/> object.</returns>
-    public static IOcelotBuilder AddPolly(this IOcelotBuilder builder) =>
-        AddPolly<PollyQoSProvider>(builder, DefaultDelegatingHandler, DefaultErrorMapping);
-
-    /// <summary>
-    /// Creates default delegating handler based on the <see cref="PollyPoliciesDelegatingHandler"/> type.
-    /// </summary>
-    /// <param name="route">The downstream route to apply the handler for.</param>
-    /// <param name="contextAccessor">The context accessor of the route.</param>
-    /// <param name="loggerFactory">The factory of logger.</param>
-    /// <returns>A <see cref="DelegatingHandler"/> object, but concreate type is the <see cref="PollyPoliciesDelegatingHandler"/> class.</returns>
-    public static DelegatingHandler DefaultDelegatingHandler(DownstreamRoute route, IHttpContextAccessor contextAccessor, IOcelotLoggerFactory loggerFactory)
-        => new PollyPoliciesDelegatingHandler(route, contextAccessor, loggerFactory);
+﻿using Microsoft.AspNetCore.Http;
```

Comment:
Follow guidances for usings, alphabetically, without spaces

Reply:
## 🤣 
This is Ray's coding style setup in JetBrains ReSharper IDE !
Ray loves to group namespace by a package predicate. And put .NET framework `System.*` namespaces at the beggining of the section.
I agree we must order namespaces always. This is the old best practice! But placing System at the beginning is new practice.